sdk/purview/azure-purview-workflow/azure/purview/workflow/_operations/_operations.py

```diff
@@ -20,8 +20,7 @@
 )
 from azure.core.paging import ItemPaged
 from azure.core.pipeline import PipelineResponse
-from azure.core.pipeline.transport import HttpResponse
-from azure.core.rest import HttpRequest
+from azure.core.rest import HttpRequest, HttpResponse
```

Comment:
Since there is multi operation groups, the `_operations` folder is renamed to `operations`. Please remove useless `_operations` folder manually.

Reply:
Removed, thanks.